src/library-authoring/LibraryLayout.tsx

```diff
@@ -24,14 +25,20 @@ const LibraryLayout = () => {
   }
 
   const navigate = useNavigate();
-  const goBack = React.useCallback(() => {
-    // Go back to the library
-    navigate(`/library/${libraryId}`);
+  const goBack = React.useCallback((prevPath?: string) => {
+    if (prevPath) {
+      // Redirects back to the previous route like collection page or library page
+      navigate(prevPath);
```

Comment:
```suggestion
      navigate(-1);
```
If you want to go back, it's not necessary to pass `prevPath` through all these callbacks like you've done here. You can just call `navigate(-1)`. I actually had this in an earlier PR, but @rpenido [suggested changing it](https://github.com/openedx/frontend-app-authoring/pull/1240#discussion_r1752936565).

For now I don't really care either way because what we really need to do is refactor this to display the editor as a modal, instead of navigating to a new URL for the editor and then going back. Obviously that will have to come in a dedicated PR though.